packages/core/upload/server/services/metrics/index.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const getProviderName = () => strapi.config.get('plugin.upload.provider', 'local');
+const getProviderIsPrivate = () => strapi.plugin('upload').provider.isPrivate();
+
+module.exports = ({ strapi }) => ({
+  async sendUploadPluginMetrics() {
+    const provider = await getProviderName();
+    const isPrivate = await getProviderIsPrivate();
```

Comment:
```suggestion
    const provider = getProviderName();
    const isPrivate = getProviderIsPrivate();
```

Reply:
isPrivate is defined as async in the upload provider. (we wrap the upload provider methods to be async) , if it's ok I will change only `getProviderName`